pkg/cockroach/cockroach.go

```diff
@@ -64,3 +65,33 @@ func BuildURI(params map[string]string) (string, error) {
 		u, h, p, db, an, ssl, dir, dir, u, dir, u,
 	), nil
 }
+
+// GetVersion returns the Schema Version of the requested DB Name
+func GetVersion(ctx context.Context, db *DB, dbName string) (string, error) {
+	query := fmt.Sprintf(`
+		SELECT EXISTS (
+  		SELECT *
+		  FROM information_schema.tables 
+		WHERE table_name = 'schema_versions'
+		AND table_catalog = '%s'
+   )`, dbName)
```

Comment:
nit: We should probably pass the database name in using SQL parameters (i.e., [$1](https://github.com/interuss/dss/blob/0f151b572a5711e999f7283e55b4b1f4c3189a65/pkg/rid/store/cockroach/identification_service_area.go#L87))